src/lib/blkalloc/append_blk_allocator.h

```diff
@@ -109,7 +109,7 @@ class AppendBlkAllocator : public BlkAllocator {
     std::string to_string() const override;
 
     void cp_flush(CP* cp) override;
-
+    void recovery_completed() override {}
```

Comment:
Needed to be explicit notification that recovery is completed, so that it can rebuild the caches (for certain blk allocators)